features/networking/service.feature

```diff
@@ -153,3 +153,433 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-24668
+  @admin
+  Scenario: externalIP defined in service but no sepc.externalIP defined	
+    Given I switch to the first user
```

Comment:
this step should not be useful since only one user is ok for this case

Reply:
Remove it based on your comment